src/User32/User32+CURSORINFOFlags.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="CURSORINFOFlags"/> nested type.
+    /// </content>
+    public static partial class User32
+    {
+        /// <summary>
+        /// Flags for the <see cref="CURSORINFO.flags" /> field.
+        /// </summary>
+        public enum CURSORINFOFlags
```

Comment:
I think it returns -1 if there is no mouse on the system.
Maybe add a `NO_MOUSE = -1` but I don't know win32 well.

Reply:
If you can find docs for that, I'm happy to add. These are based on https://docs.microsoft.com/en-us/windows/desktop/api/Winuser/ns-winuser-tagcursorinfo